packages/venia-ui/lib/components/Wishlist/WishlistDialog/wishlistDialog.js

```diff
@@ -0,0 +1,117 @@
+import React, { Fragment, useCallback, useMemo } from 'react';
+import { useIntl } from 'react-intl';
+import { Relevant } from 'informed';
+
+import { useWishlistDialog } from '@magento/peregrine/lib/talons/Wishlist/WishlistDialog/useWishlistDialog';
+
+import Dialog from '@magento/venia-ui/lib/components/Dialog';
+import { mergeClasses } from '@magento/venia-ui/lib/classify';
+import NewWishlistForm from './NewWishlistForm/newWishlistForm';
+import FormError from '../../FormError';
+import WishlistLineItem from './wishlistLineItem';
+
+import defaultClasses from './wishlistDialog.css';
+
+const WishlistDialog = props => {
+    const { isOpen, onClose } = props;
+    const classes = mergeClasses(defaultClasses, props.classes);
+
+    const talonProps = useWishlistDialog({
+        itemOptions: props.itemOptions,
+        onClose
+    });
+
+    const {
+        canCreateWishlist,
+        formErrors,
+        handleAddToWishlist,
+        handleCancel,
+        handleNewListClick,
+        handleCancelNewList,
+        isAddLoading,
+        isFormOpen,
+        wishlistsData
+    } = talonProps;
+
+    const { formatMessage } = useIntl();
+
+    const createButtonText = formatMessage({
+        id: 'wishlistDialog.createButton',
+        defaultMessage: '+ Create a new list'
+    });
+
+    const maybeListsElement = useMemo(() => {
+        if (wishlistsData) {
+            const wishlists = wishlistsData.customer.wishlists.map(wishlist => {
+                const name = `"${wishlist.name}"`;
```

Comment:
I know this is in the mocks but what do we think about dropping the quotes around the wishlist names?

Reply:
Oh Andy please stop asking me to change things already approved by @schensley 😆